sdk/nodejs/resource.ts

```diff
@@ -77,17 +75,17 @@ export abstract class Resource {
      */
     constructor(t: string, name: string, custom: boolean, props: Inputs = {}, opts: ResourceOptions = {}) {
         if (!t) {
-            throw new RunError("Missing resource type argument");
+            throw new Error("Missing resource type argument");
```

Comment:
notes:
1. these should be *not* be RunErrors.  If we fail to generate a resource, we want to give hte user a stack in their app so they know where the problem came from.  Otherwise, if you just got "missing resource name", it would be totally unhelpful as to where they'd need to fix things up.